tests/features/event.feature

```diff
@@ -6,7 +6,7 @@ Feature: Event content creation
 
   @javascript
   Scenario: Fields on the event content creation forms should be grouped logically.
-    Given I am logged in as a user with the "create oe_event content, access content, edit own oe_event content, view published skos concept entities, administer event venue entities, administer event profile entities" permission
+    Given I am logged in as a user with the "create oe_event content, access content, edit own oe_event content, view published skos concept entities, manage custom content entities" permission
```

Comment:
Question here:
```
Given I am logged in as a user with the "create oe_event content, access content, edit own oe_event content, view published skos concept entities, manage custom content entities" permission
```

Do users need to have the `manage custom content entities` permission in order to create corporate entities via an inline form on the event content type? If so we might need to break down that catch all permission into separate ones